src/components/Header/Header.scss

```diff
@@ -0,0 +1,45 @@
+.head {
+  align-items: center;
+  background: #f5f6f7;
+  border-bottom: solid 1px #dfe0e1;
+  display: flex;
+  line-height: 30px;
+  padding: 5px;
```

Comment:
make this `0 20px`; you don't need the vertical padding if you have `align-items: center`, and eventually we'll need the menu on the left to touch the top/bottom